L/log4cplus/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "log4cplus"
+version = v"2.0.8"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/log4cplus/log4cplus/releases/download/REL_$(version.major)_$(version.minor)_$(version.patch)/log4cplus-$version.tar.gz", "cdc3c738e00be84d8d03b580816b9f12628ecc1d71e1395080c802615d2d9ced")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/log4cplus-*
+
+./configure \
+--prefix=${prefix} \
+--build=${MACHTYPE} \
+--host=${target} \
+--enable-tests=no
+
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("liblog4cplusU", :liblog4cplusU),
+    LibraryProduct("liblog4cplus", :liblog4cplus)
```

Comment:
```julia
julia> Base.BinaryPlatforms.parse_dl_name_version("liblog4cplus-2-0-3.dll", "windows")[1]
"liblog4cplus-2-0"

julia> Base.BinaryPlatforms.parse_dl_name_version("liblog4cplusU-2-0-3.dll", "windows")[1]
"liblog4cplusU-2-0"
```
```suggestion
    LibraryProduct(["liblog4cplusU", "liblog4cplusU-2-0"], :liblog4cplusU),
    LibraryProduct(["liblog4cplus", "liblog4cplus-2-0"], :liblog4cplus),
```

Reply:
Do you have any idea where the `2-0-3` is coming from? That seems like a typo error upstream or something? I would have guessed `2-0-8`? I searched around but could not find anything.

Reply:
Maybe https://github.com/log4cplus/log4cplus/blob/52dd360e04d0fcdadc87cc1874884d9bbc027f99/configure.ac#L7, not entirely sure (also because I have no idea what `3.0.0` refers to).